src/helm/benchmark/static/schema.yaml

```diff
@@ -2116,6 +2116,66 @@ run_groups:
       main_name: f1_score
       main_split: test
 
+  - name: billsum_legal_summarization
+    display_name: BillSum
+    description: The BillSum benchmark for legal text summarization ([Kornilova & Eidelmann, 2020](https://aclanthology.org/D19-5406/)).
+    metric_groups:
+      - accuracy
+      - summarization_metrics
+      - bias
+      - toxicity
+      - efficiency
+      - general_information
+    environment:
+      main_name: rouge_2
+      main_split: test
+    taxonomy:
+      task: summarization
+      what: "?"
```

Comment:
Could you fill these in?  (We need to start doing this for the entire codebase.)

Reply:
Sure, just pushed an update.